### ISSUE TITLE
Upgrade base VScode to latest version

### ISSUE THREAD

**Issue by fisforfaheem** [getcursor/cursor#1896]
Cursor is using a very old version on BASE VSCODE please use latest and support new TECH

**Comment by DaiZack** [comment]
In the old vscode 1.91.1 there is a pylance 2024.8.1 bug that when using interactive window https://github.com/microsoft/pylance-release/issues/6370 there is a fix for it 2024.9.1 on pylance but requires newer version of vscode.

**Comment by fisforfaheem** [comment]
Yes, we really need the base update

**Comment by ultimateshadsform** [comment]
Yes, I have 2 extensions that require a newer version of vscode. I need ^1.93.0 at least.

**Comment by markNZed** [comment]
I assume Cursor uses Theia not VS Code

**Comment by danperks** [comment]
I believe Cursor uses the open source "Code" project, which is what the proprietary VSCode app uses.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
